pkgs/by-name/ra/rainfrog/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+}:
+let
+  version = "0.2.3";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "rainfrog";
+
+  src = fetchFromGitHub {
+    owner = "achristmascarl";
+    repo = "rainfrog";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
It became mandatory?

Reply:
It's highly recommended as it prevents branch-tag mismatches.